hadoop-tools/hadoop-azure/src/main/java/org/apache/hadoop/fs/azurebfs/AzureBlobFileSystem.java

```diff
@@ -511,8 +512,10 @@ public FSDataOutputStream createNonRecursive(final Path f, final FsPermission pe
     String parentPath = parent.toUri().getPath();
     if (getAbfsStore().getAbfsConfiguration().getPrefixMode() == PrefixMode.BLOB
```

Comment:
Can be directly getAbfsStore().getPrefixMode()

Reply:
taken.